clinvar_parser/clinvar-parser/src/main/java/uk/ac/ebi/eva/clinvar/PublicSetParser.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi.eva.clinvar;
+
+import uk.ac.ebi.eva.clinvar.model.ClinvarSet;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.transform.stream.StreamSource;
+import java.io.StringReader;
+
+class PublicSetParser {
+
+    Unmarshaller unmarshaller;
```

Comment:
Is used just in the ClinvarSetTransformer class. I cannot do it private, but I could make it an inner class of ClinvarSetTransformer. The same happens with XmlClinVarReader and XmlClinvarSetIterator. 

Maybe  a good solution would be create some "read", "transform" and "write" packages and move the classes there (with the auxiliar ones having package visibility). That would make writting tests for those auxiliar classes easier. What do you think?